driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -243,6 +243,38 @@ public static Bson lookup(final String from, final String localField, final Stri
                                                    .append("as", new BsonString(as)));
     }
 
+    /**
+     * Creates a $lookup pipeline stage for the specified filter
```

Comment:
Let's fix this comment now, as there is no `filter`, but rather a pipeline.  Perhaps:

   Creates a $lookup pipeline stage, joining the current collection with the one specified in `from` using the given `pipeline`.

Reply:
(I realize the comment in the existing lookup method is misleading as well.  Feel free to fix that one too if you're so inclined.)